Tests/test_GenBank.py

```diff
@@ -187,19 +186,17 @@ def t_write_format():
 def t_cleaning_features():
     """Test the ability to clean up feature values.
     """
-    parser = GenBank.FeatureParser(feature_cleaner=utils.FeatureValueCleaner())
+    gb_parser = GenBank.FeatureParser(feature_cleaner=utils.FeatureValueCleaner())
     handle = open(os.path.join("GenBank", "arab1.gb"))
-    iterator = GenBank.Iterator(handle, parser)
+    iterator = GenBank.Iterator(handle, gb_parser)
 
     first_record = next(iterator)
 
     # test for cleaning of translation
     translation_feature = first_record.features[1]
     test_trans = translation_feature.qualifiers["translation"][0]
-    assert ' ' not in test_trans, \
-      "Did not clean spaces out of the translation"
-    assert '\012' not in test_trans, \
-      "Did not clean newlines out of the translation"
+    assert ' ' not in test_trans, "Did not clean spaces out of the translation"
+    assert '\012' not in test_trans, "Did not clean newlines out of the translation"
```

Comment:
I should have said aim for 79 characters as per PEP8, https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Running the ``pep8`` tool locally would be helpful here.